src/fault-handler.jl

```diff
@@ -1,4 +1,8 @@
-"Recursively checks if an exception was caused by a worker exiting."
+"""
+    check_exited_exception(res) -> Bool
```

Comment:
This is technically an internal function only used by the fault handler, but I think this is still a fine change. Maybe just add `res::Exception` to the argument to indicate that it takes an `Exception` as input?